src/AutoDelete.sh

```
@@ -0,0 +1,21 @@
+main() {
+
+    export PATH=$PATH:/home/dnanexus/.local/bin  
+
+    sudo -H python3 -m pip install --no-index --no-deps packages/*
+    
+    dx download "$config_file" -o config_file
+
+    python3 /home/dnanexus/eggd_automatic_deletion/main.py -c config_file
+
+
+if [[ -s /home/dnanexus/*_files_to_delete_*.csv ]]; then
```

Shellcheck (0.10.0)

[error] -s doesn't work with globs. Use a for loop.

(SC2144)